presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcType.java

```diff
@@ -282,4 +282,9 @@ public String toString()
 
         return orcTypes;
     }
+
+    public static OrcType createOrcType(OrcTypeKind orcTypeKind)
+    {
+        return new OrcType(orcTypeKind);
```

Comment:
OrcTypeKind is not enough to create a valid instance of OrcType. This method is only used in a test, hence, I suggest to remove it and update the caller like so:

```
    private static StreamDescriptor makeStreamDescriptor(int levels)
    {
        DummyOrcDataSource orcDataSource = new DummyOrcDataSource();

        OrcType intType = new OrcType(INT, ImmutableList.of(), ImmutableList.of(), Optional.empty(), Optional.empty(), Optional.empty());
        OrcType listType = new OrcType(LIST, ImmutableList.of(1), ImmutableList.of("item"), Optional.empty(), Optional.empty(), Optional.empty());

        StreamDescriptor streamDescriptor = new StreamDescriptor("a", 0, "a", intType, orcDataSource, ImmutableList.of());
        for (int i = 0; i < levels; i++) {
            streamDescriptor = new StreamDescriptor("a", 0, "a", listType, orcDataSource, ImmutableList.of(streamDescriptor));
        }

        return streamDescriptor;
    }
```